src/sentry/models/savedsearch.py

```diff
@@ -52,6 +68,9 @@ class SavedSearch(Model):
     is_default = models.BooleanField(default=False)
     is_global = models.NullBooleanField(null=True, default=False, db_index=True)
     owner = FlexibleForeignKey("sentry.User", null=True)
+    visibility = CharField(
+        max_length=16, default=Visibility.OWNER, choices=Visibility.as_choices(), null=True
```

Comment:
We probably want to change the frontend to send `visibility: 'org'` before merging this, right? Otherwise saved searches made will be made with `visibility: 'owner'` which isn't quite a thing yet.